src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -60,13 +60,15 @@ internal static class CodeActionHelpers
                     {
                         if (!IsCodeActionNotSupportedByLSP(suggestedAction))
                         {
+                            using var _1 = ArrayBuilder<string>.GetInstance(out var codeActionPath);
```

Comment:
```suggestion
                            using var _1 = ArrayBuilder<string>.GetInstance(out var codeActionPathList);
```

(let's name things uniformly everywhere).